package/com.unity.formats.usd/Dependencies/USD.NET/serialization/TypeBinder.cs

```diff
@@ -429,15 +429,21 @@ object GuidToCs_Bytes(pxr.VtValue vtValue)
             return ret;
         }
 
+        /*
```

Comment:
if this block is commented, can we just remove it?

Reply:
I'm reluctant to remove it in case it's needed in future, as it appears to have been added to support GUIDs and while I don't think we should need support for GUID parsing, there could be a usecase I've missed. This relates to the comment I added on lines 480 onwards :)